src/utils.rs

```diff
@@ -79,15 +79,16 @@ pub fn file_exists<P: AsRef<Path>>(path: P) -> bool {
 
 pub fn get_string(label: &str, config: &Config) -> Result<String> {
     let mut file = try!(fs::File::open({
-        let path = format!("{}/{}/res/values-en/strings.xml",
-                           config.get_dist_folder(),
-                           config.get_app_id());
-        if file_exists(&path) {
+        let path = config.get_dist_folder()
+            .join(config.get_app_id())
+            .join("res/values-en/strings.xml");
```

Comment:
Will this work multiplatform?

Also, not sure if using `lazy_static!` to initialize the path would make a difference, probably not, since we need to open the file once and again. We could open it only once, though, and store all strings in memory, and search always in memory. Not really a revew ofyour code, but something to think about in future changes to this function.